controllers/src/main/java/com/redhat/ipaas/controllers/integration/online/DeactivateHandler.java

```diff
@@ -53,7 +54,13 @@ public StatusUpdate execute(Integration integration) {
             ? Integration.Status.Deactivated
             : Integration.Status.Pending;
 
-        openShiftService.scale(deployment);
+        try {
+            openShiftService.scale(deployment);
+        } catch (KubernetesClientException e) {
+            if( !"Resource not found!".equals(e.getMessage()) ) {
```

Comment:
yes, please change. never ever put business meaning on exception messages. Imagine the upstream developer improving the exception by including which kind of resource was not found. I as an upstream would never think of someone matching on my exception messages ...